src/App.js

```diff
@@ -1,30 +1,41 @@
 import React, { Component } from 'react';
 import MovieContainer from './MovieContainer';
 import './App.css';
-import movieData from './MovieData';
 import Header from './Header';
 import SelectedMovie from './SelectedMovie';
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies
+      movies: [],
+      currentSection: 'mainPage',
+      selectedID: 0,
+      error: ''
     }
   }
 
-  componentDidMount() {
+  componentDidMount = () => {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
Free free to move this fetch into a separate file just like y'all did in Mod 2. It's still common to separate items like that out in the React world and you should be able to import them just like before.